.github/workflows/ci-build.yml

```
@@ -0,0 +1,48 @@
+
+name: CRDA CI Build
+
+on:
+  pull_request:
+    branches: [ master ]
+   
+jobs:
+  Linter-docstyle:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: 
+        scripts: ["run-linter", "check-docstyle", "detect-dead-code", "detect-common-errors"]
```

Use yaml style of array declaration.
```suggestion
        scripts: 
          - run-linter
          - check-docstyle
          - detect-dead-code
          - detect-common-errors
```